src/Router.php

```diff
@@ -84,4 +84,23 @@ class Router {
         throw new NotFoundRedirectException();
     }
   }
+
+  public static function getRequestedPage(): string {
+    $page = idx(Utils::getGET(), 'page') ?: idx(Utils::getGET(), 'p');
+    if (!is_string($page)) {
+      $page = 'index';
+    }
+
+    return strval($page);
+  }
+
+  public static function isRequestAjax(): bool {
+    return Utils::getGET()->get('ajax') === 'true';
+  }
+
+  // Check to see if the request is going through the router
+  public static function isRequestRouter(): bool {
+    if (self::getRequestedPage() == "index") return false;
```

Comment:
We will update this.  I'm not sure why the shorthand versions were used in some cases and the longer form in others.  We will go back over the code and see if there are any other similar cases.

As for the Router::isRequestRouter() method, we're using the Router::getRequestedPage() method to retrieve the "p" or "page" request variables.  For simplicity, if neither 'p' nor 'page' are set, we are returning a default of "index."  None of the non-router requests have either the "p" or "page" variables set, so we are then determining the request did not go through the Router.

However, this would be inaccurate on an index.php request.  Although, in the case that it is a request directly to the root index.php, it does not impact the current usage of the methods.  We could add a check for the requested file to determine if it is a true index request.  I think such a change would be better and clarify the functionality further.